modules/cnf-reducing-netqueues-using-pao.adoc

```diff
@@ -0,0 +1,347 @@
+<5>  All devices that have an interface name `eth0`, `vendorID` 0x1af4 and `deviceID` 0x1000.
+
+. Apply the performance profile.
++
+[source,terminal]
+----
+$  oc apply -f <your-profile-name>.yaml
+----
+
+[id="verify-netqueues-changes_{context}"]
+== Verify the queue status
+
+In this section a number of examples illustrate different performance profiles and how to verify the changes are applied.
+
+.Example 1
+
+In this example the net queue count is set to the reserved CPU count (two) for *all* supported devices.
+
+The relevant section from the performance profile is:
+
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+metadata:
+ name: performance
+spec:
+  kind: PerformanceProfile
+  spec:
+   cpu:
+    reserved: 0-1  #total = 2
+    Isolated: 2-8
+   net:
+    userLevelNetworking: true
+    [...]
+----
+
+Display the status of the queues associated with a device using the command:
+[source,terminal]
+----
+$ ethtool -l <device>
+----
+[NOTE]
+====
+Run this command on the node where the performance profile was applied.
+====
+
+Before the profile is applied the queue status is:
+
+[source,terminal]
+----
+# ethtool -l ens4
+Channel parameters for ens4:
+Pre-set maximums:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+Current hardware settings:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+----
+After the profile is applied the queue status is:
+
+[source,terminal]
+----
+# ethtool -l ens4
+Channel parameters for ens4:
+Pre-set maximums:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+Current hardware settings:
+RX:         0
+TX:         0
+Other:      0
+Combined:   2 <1>
+----
+<1> The combined channel shows the total count of reserved CPUs for *all* supported devices two matching what is configured in the performance profile.
+
+.Example 2
+
+In this example the net queue count is set to the reserved CPU count (two) for *all* supported network devices with a specific `vendorID`.
+
+The relevant section from the performance profile is:
+
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+metadata:
+  name: performance
+spec:
+    kind: PerformanceProfile
+    spec:
+     cpu:
+      reserved: 0-1  #total = 2
+      Isolated: 2-8
+     net:
+      userLevelNetworking: true
+      devices:
+      - vendorID = 0x1af4
+    [...]
+----
+
+Display the status of the queues associated with a device using the command:
+[source,terminal]
+----
+$ ethtool -l <device>
+----
+[NOTE]
+====
+Run this command on the node where the performance profile was applied.
+====
+
+Verify the queue status after the profile is applied:
+
+[source,terminal]
+----
+# ethtool -l ens4
+Channel parameters for ens4:
+Pre-set maximums:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+Current hardware settings:
+RX:         0
+TX:         0
+Other:      0
+Combined:   2 <1>
+----
+
+<1> The total count of reserved CPUs for all supported devices with `vendorID=0x1af4` is two.
```

Comment:
In this situation, the correct format for this is `2`, as it is a specified value.